tests/devtools.test.tsx

```diff
@@ -80,18 +80,29 @@ describe('If there is no extension installed...', () => {
 })
 
 describe('When state changes...', () => {
-  it("sends { type: setStateName || 'anonymous` } as the action with current state", () => {
+  it("sends { type: testSetStateName || 'anonymous`, ...rest } as the action with current state", () => {
```

Comment:
Nitpick: We can keep this too as it is (the test is that "it sends `setStateName`" and `setStateName` in turn is `"testSetStateName"` for the given tests, but it can be anything).
```suggestion
  it("sends { type: setStateName || 'anonymous`, ...rest } as the action with current state", () => {
```